mypy/server/update.py

```diff
@@ -226,6 +228,11 @@ def update(self, changed_modules: List[Tuple[str, str]]) -> List[str]:
             changed_modules = [(id, path) for id, path in changed_modules
                                if id != next_id]
             changed_modules = dedupe_modules(remaining + changed_modules)
+            t1 = time.time()
+
+            self.manager.log_fine_grained(
```

Comment:
This seems unrelated to the rest of thePR?

Reply:
Oh, indeed. I added it to track progress while running mypy-daemon on a big changed repo. I can split it out.